### DISCUSSION TITLE
Lists do not seem to work inside examples in typst

### DISCUSSION THREAD

**Discussion by psads-git** [quarto-dev/quarto-cli#9218] [Q&A] [answered]
I am using the following `quarto` document, but getting the list wrongly displayed, as shown in the image bellow.

Could you please help me? Thanks!

```
---
format: typst
---

::: {#exm-some}

Some text:

* item 1
* item 2

:::
```

![image](https://github.com/quarto-dev/quarto-cli/assets/75945439/ead03d7c-00d4-41a8-ab78-20ef13b94a41)

**Comment by mcanouil** [comment, marked as answer]
Thanks that a bug in Quarto.

- https://github.com/quarto-dev/quarto-cli/issues/9219
